src/main/java/com/minecolonies/coremod/util/InventoryUtils.java

```diff
@@ -599,171 +527,13 @@ public static int getFirstSlotContainingTool(@NotNull final IInventory inventory
         return -1;
     }
 
-    /**
-     * Adapted from {@link net.minecraft.entity.player.InventoryPlayer#addItemStackToInventory(ItemStack)}.
-     *
-     * @param inventory Inventory to add itemstack to.
-     * @param itemStack ItemStack to add.
-     * @return True if successful, otherwise false.
-     */
-    public static boolean addItemStackToInventory(@NotNull final IInventory inventory, @Nullable final ItemStack itemStack)
+    public static boolean addItemStackToInventory(@NotNull final IItemHandler inventory, @Nullable final ItemStack itemStack)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)